src/lib/core/managers/download.svelte.ts

```
@@ -0,0 +1,388 @@
+    return streamId;
+  }
+
+  private setItemStatus(item: QueueItem, status: QueueItem['status']) {
+    item.status = status;
+    this.queue = [...this.queue];
+
+    return this.saveQueueToFile();
+  }
+
+  private async saveQueueToFile() {
+    const accountId = (await Legendary.getAccount())!;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)